test/blackbox-tests/test-cases/dialects/dots.t/run.t

```diff
@@ -0,0 +1,26 @@
+Test the (dialect ...) stanza inside the dune-project file.
+
+  $ { echo '(lang dune 3.7)'; cat dune-project.in; } >dune-project
+
+  $ dune build --display short
+  File "dune-project", line 5, characters 13-20:
+  5 |   (extension cppo.ml)
+                   ^^^^^^^
+  Error: the possibility of defining extensions with embedded dots is only
+  available since version 3.8 of the dune language. Please update your
```

Comment:
```suggestion
  available since version 3.8 of the Dune language. Please update your
```

Reply:
This message is not introduced by the present PR, so it would have to be treated elsewhere.